source/includes/fact-system-vars.rst

```diff
@@ -0,0 +1,115 @@
+.. list-table::
+   :widths: 30 10 40 10 10
+   :stub-columns: 1
+   :header-rows: 1
+
+   * - System Variable
+     - Data Type
+     - Description
+     - Default Value (Atlas-hosted)
+     - Default Value (On Premises)
+   * - ``full_pushdown_exec_mode``
+     - boolean
+     - Specifies whether a query error is returned for queries with
+       clauses that aren't fully translated to MongoDB semantics. SQL
+       query predicates are used to filter data returned by the query.
+     - ``0 (false)``
+     - ``0 (false)``
+   * - ``log_level``
+     - integer
+     - Specifies the logging level for |bi-short|:
+
+       - ``-1``: Do not log.
+       - ``0``: Log only messages for database users, such as basic
+         ``mongosqld`` events and state changes.
+       - ``1``: Log messages for database users and ``mongosqld`` administrators.
+       - ``2``: For internal use only.
+     - ``2``
+     - ``0``
+   * - ``max_nested_table_depth``
+     - integer
+     - Specifies the maximum number of unique nested field paths that
+       ``mongosqld`` maps to a relational table for a collection.
+     - ``50``
+     - ``50``
+   * - ``max_num_columns_per_table``
+     - integer
+     - The maximum number of unique fields that ``mongosqld`` maps to
+       relational columns for a collection.
+     - ``1000``
+     - ``1000``
+   * - ``mongodb_max_varchar_length``
+     - integer
+     - Specifies the maximum string length returned for columns using
+       the ``VARCHAR`` data type. A value of ``0`` specifies
+       no limit.
+     - ``0``
+     - ``0``
+   * - ``polymorphic_type_conversion_mode``
+     - string
+     - Determines how |bi-short| evaluates document fields that are
+       specified with multiple data types. For example, ``count``
+       could be a number in one document and a string in another.
+       The data type that |bi-short| selects depends on the value
+       you assign to the ``schema_mapping_mode`` system variable.
+       A value of ``majority`` causes |bi-short| to select the type
+       that appears in the majority of document fields, while a value
+       of ``lattice`` would cause |bi-short| to select string as the
+       data type for ``count``.
```

Comment:
I'd link to https://docs.mongodb.com/bi-connector/master/schema/type-conflicts/ for additional context.

Reply:
Fixed.